src/jdk.jpackage/share/classes/jdk/jpackage/internal/cli/StandardOption.java

```diff
@@ -0,0 +1,808 @@
+            .createArray(toList());
+
+    public static final OptionValue<List<Path>> MODULE_PATH = pathOption("module-path").addAliases("p").tokenizer(pathSeparator())
+            .valuePattern("module path")
+            .outOfScope(NOT_BUILDING_APP_IMAGE)
+            .mutate(createOptionSpecBuilderMutator((b, context) -> {
+                if (context.os() == OperatingSystem.WINDOWS) {
+                    b.description("main.option.module-path" + resourceKeySuffix(context.os()));
+                }
+            }))
+            .createArray(toList());
+
+    public static final OptionValue<Boolean> LAUNCHER_AS_SERVICE = booleanOption("launcher-as-service")
+            .scope(nativeBundling())
+            .inScope(LauncherProperty.VALUE)
+            .create();
+
+    //
+    // Linux-specific
+    //
+
+    public static final OptionValue<String> LINUX_RELEASE = stringOption("linux-app-release").scope(nativeBundling()).create();
+
+    public static final OptionValue<String> LINUX_PACKAGE_NAME = stringOption("linux-package-name")
+            .valuePattern("package name")
+            .scope(nativeBundling()).create();
+
+    public static final OptionValue<String> LINUX_DEB_MAINTAINER_EMAIL = stringOption("linux-deb-maintainer")
+            .valuePattern("email address")
+            .create();
+
+    public static final OptionValue<String> LINUX_APP_CATEGORY = stringOption("linux-app-category").scope(nativeBundling()).create();
+
+    public static final OptionValue<String> LINUX_RPM_LICENSE_TYPE = stringOption("linux-rpm-license-type")
+            .valuePattern("license type")
+            .scope(nativeBundling()).create();
+
+    public static final OptionValue<String> LINUX_PACKAGE_DEPENDENCIES = stringOption("linux-package-deps")
+            .valuePattern("package-dep-string")
+            .scope(nativeBundling()).create();
+
+    public static final OptionValue<LauncherShortcut> LINUX_SHORTCUT_HINT = launcherShortcutOption("linux-shortcut")
+            .scope(nativeBundling())
+            .inScope(LauncherProperty.VALUE)
+            .create();
+
+    public static final OptionValue<String> LINUX_MENU_GROUP = stringOption("linux-menu-group")
+            .valuePattern("menu-group-name")
+            .scope(nativeBundling()).create();
+
+    //
+    // MacOS-specific
+    //
+
+    public static final OptionValue<List<Path>> MAC_DMG_CONTENT = pathOption("mac-dmg-content")
+            .valuePattern("additional content path")
+            //.toArray(pathSeparator())
+            .tokenizer(",")
+            .createArray(toList());
+
+    public static final OptionValue<Boolean> MAC_SIGN = booleanOption("mac-sign").scope(MAC_SIGNING).addAliases("s").create();
+
+    public static final OptionValue<Boolean> MAC_APP_STORE = booleanOption("mac-app-store")
+            //.scope(MAC_SIGNING) // TODO: --mac-app-store should be applicable to app image signing operation because it redefines signing key
```

Comment:
I do not like any TODO comments. They usually never got fixed. Bug needs to be filed if follow up is required.